src/index.html

```diff
@@ -1,13 +1,361 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./images/favicon.png"
+    >
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="header__top">
+        <a href="index.html">
+          <img src="./images/logo-white.svg"
+            alt="logo"
+            class="header__logo logo"
+          >
+        </a>
+
+        <div class="header__controls">
+          <div class="header__wrapper-phone">
+            <a href="tel:+12345555555"
+              class="header__phone-number">
+              +1 234 555-55-55
+            </a>
```

Comment:
https://github.com/mate-academy/layout_landing-page/assets/106310796/18d2f6b0-4999-4d43-bd7f-0f4ae30c0e06

Reply:
transition should work both ways